starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,43 @@
+
+* {
+  font-family: 'Open Sans', sans-serif; 
+  font-size: 30px;
+}
+
+.header {
+  background-image: url(file:///home/looffi/code/labs/lab-css-recipes-clone/starter-code/apple-pie-starter-code/images/apple-pie.jpg);
+  width: 100%;
+  background-size: cover;
+  height: 700px;
+}
+
+h1 {
+  font-size: 80px;
+  color: white;
+  font-weight: 500;
+  text-align: center;
+  padding-top: 350px; 
```

Comment:
Here, adding some `padding-top` is not a very good solution because because it will not scale with different image sizes. There's one very precise way of doing it: `translate`.

Here's what your code could look like:
```css
h1 {
  font-size: 80px;
  color: white;
  font-weight: 500;
  text-align: center;
  position: relative;
  top: 50%;
  transform: translateY( -50% );
}
```

This technique also works very well for other element centering inside containers